app/Http/Transformers/PostTransformer.php

```diff
@@ -66,6 +66,7 @@ public function transform(Post $post)
             $response['source_details'] = $post->source_details;
             $response['remote_addr'] = '0.0.0.0';
             $response['details'] = $post->details;
+            $response['school_id'] = $post->school_id;
```

Comment:
Does this need to be added to the c.io payload as well?

Reply:
Doesn't seem to hurt sending over while we're here.. @ngjo and I were chatting over whether we need this (and the perhaps-more-helpful school name) over in https://www.pivotaltracker.com/story/show/169745644/comments/208852773

Reply:
Ah yeah I guess school name is probably more helpful for c.io, but like you said, id can't hurt